osu.Game/Screens/Play/Player.cs

```diff
@@ -684,11 +692,9 @@ private void onFailComplete()
             LoadedBeatmapSuccessfully && Configuration.AllowPause && ValidForResume
             // replays cannot be paused and exit immediately
             && !DrawableRuleset.HasReplayLoaded.Value
+            // cannot pause if we are already in a fail state
```

Comment:
realized this got removed by the drastic changes of the conditionals, added it back here.